functions-common

```diff
@@ -312,6 +312,9 @@ declare -g os_VENDOR os_RELEASE os_PACKAGE os_CODENAME
 # user if not available.  Note can't use generic install_package*
 # because they depend on this!
 function _ensure_lsb_release {
+    # ignore lsb_release check
+    return
```

Comment:
Does Clear have anything similar to this at all?  We use it to get the values returned in GetOSVersion().  We can probably detect Clear by looking for swupd and simulating an equivalent for what we need.

Reply:
We can use "grep "Clear Linux" /lib/os-release".
I will try to make it work today.